### PR TITLE
support for setting node region/zone on creation

### PR DESCRIPTION
**Pull request by detiber**
- Add ansible-aws subdirectory for beta4
- Update beta4 ansible inventory for setting regions/zones
- Some formatting fixes for both beta-3-setup and beta-4-setup
- add region/zone setting to ansible-aws openshift_setup playbook

### CODE REVIEW

**Comment by detiber** [comment]
@thoraxe the start of docs for region/zone support in openshift-ansible

**Comment by thoraxe** [comment]
I think this needs a rebase. I am noticing a lot of header/style changes that look like you didn't catch or accidentally didn't get merged on your side.

**Comment by detiber** [comment]
@thoraxe There is still a pretty large diff in the beta4 doc, but it the only changes I made were for consistency in the way we are doing blockquoting and fixing an error that was causing doctoc to not parse all of the sections correctly, I also changed the header level of some of the sections underneath the IaaS appendix. Otherwise the TOC changes were purely from running doctoc --maxlevel 3

File: beta-3-setup.md [hunk 1]
```diff
@@ -3641,30 +3626,30 @@ configure the entire AWS environment, too.
 
 ## Generic Cloud Install
 
-### An Example Hosts File (/etc/ansible/hosts)
+#### An Example Hosts File (/etc/ansible/hosts)
```
Comment (thoraxe) [inline beta-3-setup.md]: Why are we using level 4 headings here?

**Comment by thoraxe** [comment]
I double checked and the level 4 headings are really the only issue it seems.

**Comment by thoraxe** [comment]
Well, that and I will have to figure out how to cherry-pick merge the beta3 doc changes from the beta4 branch back into the master branch... :-/

**Comment by detiber** [comment]
@thoraxe I just did the level 4 headings to prevent them from showing up in the TOC, if you think they should be there, then I can switch back to level 3.  Also, I can handle the beta3 doc changes as a separate PR.

**Comment by thoraxe** [comment]
If you don't think they should be in the TOC, why are they headings? :)

**Comment by detiber** [comment]
@thoraxe what headings? :smile:

**Comment by thoraxe** [comment]
I think this goes a little far in the other direction, but I'll take it :)

**Pull request merged**